cfg_mgmt/model.py

```diff
@@ -180,6 +180,17 @@ class CfgMetadata:
     queue: list[CfgQueueEntry]
 
 
+@dataclasses.dataclass
+class CfgStatusAnalysis:
```

Comment:
I think the name is misleading. This is the _result_ of an analysis, maybe, but does not represent the analysis itself. Naming seems a bit difficult here, since we already have a `CfgReportingSummary`. How about `CfgStatusEvaluation(Result)`? Also, a brief docstr explaining the interrelationship between CfgStatus and the (derived) CfgStatusEvaluation(Result) might help.